templates/registration/logged_out.html

```diff
@@ -0,0 +1,8 @@
+{% extends "base.html" %}
+
+{% block content %}
+  
+  <p>Logged out!</p>
+  <a href="{% url 'login'%}">Click here to login again.</a>
```

Comment:
Consider adding some spacing between the 'login' and the closing '%}' in the URL tag for better readability.